tests/Integration/Behaviour/Features/Scenario/Contact/contact.feature

```diff
@@ -0,0 +1,27 @@
+@reset-database-before-feature
+Feature: Contact
+  In order to create customizable contact us form for customers
+  As a BO user
+  I should be able to add and edit new contact
+
+  Scenario: Add new contact
+    Given there is no contact with id 3
+    And there is contact with id 2
```

Comment:
I'm worried about relying on ID generation strategy 🤔
- it's not very user-oriented (customers will talk about contact "Rokas Z" or "Mathieu F" rather than contact "with ID 34")
- if one day we change the IDs generation strategy (for example to use UUIDs) all these tests will crash although the logic is still good

@PierreRambaud @jolelievre Any opinion ?
I guess we could start creating dedicated DataProviders / Repositories with search functions `getContactByName()`, `lookForContact()` and use them to find the IDs needed by commands and queries ?

Reply:
I agree the tests should rely on database IDs But maybe you don't need a repository In other behat context we use the `SharedStorage` to store references to objects Then you can get the id from the referenced saved object

Reply:
Changed id generation strategy to references and SharedStorage in the last commit